sunpy/instr/aia/read_genx.py

```diff
@@ -0,0 +1,386 @@
+from __future__ import (absolute_import, division, print_function,
```

Comment:
That makes a lot of sense. My point really was is calling this file `read_genx` a little to general, seeing how it's aia specific.